Week5/week5solution1.ipynb

```diff
@@ -0,0 +1,156 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "91d973b0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import csv"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "83e78ad4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "a=['sid','sname','city','contact']"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "6770c868",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "records=[[1,'raj','bharuch',9632591232],\n",
+    "         [2,'ved','bardoli',9785785429],\n",
+    "         [3,'om','surat',8485963218],\n",
+    "         [4,'ved','baroda',9863521478],\n",
+    "         [5,'trith','valod',9930452161]]\n",
+    "p=[]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "75f8ae18",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "enter student id:6\n",
+      "enter student name:jay\n",
+      "enter student city:surat\n",
+      "enter student contact number:8965321478\n",
+      "enter student id:7\n",
+      "enter student name:vikas\n",
+      "enter student city:bardoli\n",
+      "enter student contact number:7896541239\n",
+      "enter student id:8\n",
+      "enter student name:jugal\n",
+      "enter student city:valod\n",
+      "enter student contact number:9563214785\n",
+      "enter student id:9\n",
+      "enter student name:denil\n",
+      "enter student city:mandvi\n",
+      "enter student contact number:7894563259\n",
+      "enter student id:10\n",
+      "enter student name:hiren\n",
+      "enter student city:valod\n",
+      "enter student contact number:9632145879\n"
+     ]
+    }
+   ],
+   "source": [
+    "for i in range(5):\n",
+    "    sid=int(input(\"enter student id:\"))\n",
+    "    sname=input(\"enter student name:\")\n",
+    "    city=input(\"enter student city:\")\n",
+    "    contact=int(input(\"enter student contact number:\"))\n",
+    "    l=[sid,sname,city,contact]\n",
+    "    p.append(l)"
```

Comment:
why you use append() ??

Reply:
add items to the end of an existing list object